sql/moz-fx-data-shared-prod/mozilla_vpn_derived/site_metrics_summary_v1/query.sql

```diff
@@ -88,7 +85,12 @@ base_table AS (
       STRUCT(campaign, content, medium, source)
     ).*,
   FROM
-    ga_sessions_hits
+    ga_sessions
+  CROSS JOIN
+    UNNEST(hits)
+  WHERE
+    `date` IS NOT NULL
+    AND `date` = CURRENT_DATE - 1
```

Comment:
Why did this change from `@date` to `CURRENT_DATE - 1`?

Reply:
copy pasta >_<